src/admin/components/forms/field-types/RichText/elements/textAlign/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ElementButton from '../Button';
+import AlignLeftIcon from '../../../../../icons/AlignLeft';
+import AlignCenterIcon from '../../../../../icons/AlignCenter';
+import AlignRightIcon from '../../../../../icons/AlignRight';
+
+export default {
+  name: 'alignment',
```

Comment:
What do you think about renaming this to `textAlign`? Just so that everything is consistent

Reply:
can do!